### PR TITLE
change Python license headers to be # comments instead of docstrings

### PR DESCRIPTION
**Pull request by sherifnada**
For the CDK we want to autogenerate API Reference docs based on class/method docstrings. Because LICENSE is currently added as a docstring to the top of every python file the LICENSE header gets mistaken as something that should be included in reference docs. 

The only file of interest is `build.gradle`. All other files have identical changes to the license header.

### CODE REVIEW

**Review by davinchia — APPROVED** [review]

File: airbyte-integrations/bases/airbyte-protocol/airbyte_protocol/__init__.py [hunk 1]
```diff
@@ -1,28 +1,3 @@
-"""
```
Comment (jrhizor) [inline airbyte-integrations/bases/airbyte-protocol/airbyte_protocol/__init__.py:1]: I'm a bit late for this, but why are these removed instead of transformed into the # comments?

File: airbyte-integrations/bases/airbyte-protocol/airbyte_protocol/models/__init__.py [hunk 2]
```diff
@@ -1,27 +1,2 @@
-"""
```
Comment (jrhizor) [inline airbyte-integrations/bases/airbyte-protocol/airbyte_protocol/models/__init__.py:1]: same here

Comment (sherifnada) [reply airbyte-integrations/bases/airbyte-protocol/airbyte_protocol/models/__init__.py:1]: I excluded `__init__.py` files because a lot of them were blank and I wasn't sure how to write a regex that made spotless realize that. I could add a bogus comment or import I guess, but since they were blank it didn't seem important?
